src/common/src/array/list_array.rs

```diff
@@ -612,6 +612,24 @@ impl Debug for ListRef<'_> {
     }
 }
 
+impl Row for ListRef<'_> {
```

Comment:
Is this a little confusing 🤔

Reply:
This is to allow arguments of list type to be accepted as `impl Row` in Rust functions.

Reply:
I'm okay with this as I cannot come up with another way to interpret `ARRAY` as rows to form an ambiguousness. If there's one, making a newtype on `ListRef` then `impl Row` on that will work.